starter-code/javascript/chronometer.js

```diff
@@ -1,27 +1,39 @@
 // Constructor
-// function Chronometer() {
-
-// }
-
-// Chronometer.prototype.startClick = function () {
-
-// };
-
-// Chronometer.prototype.setMinutes = function () {
-  
-// };
-
-// Chronometer.prototype.setSeconds = function () {
-  
-// };
-
-// Chronometer.prototype.twoDigitsNumber = function () {
-  
-// };
-
-// Chronometer.prototype.setTime = function () {
-
-// };
+function Chronometer() {
+    this.currentTime = 0;
+    this.intervalId = 0;
+};
+
+
+Chronometer.prototype.startClick = function () {
+    var self = this;
+    window.setInterval(function() {
```

Comment:
You don't need window here. But you need to store the interval in a variable for being able to clean it later:
```
self.intervalId = setInterval(function () {
```